app/src/main/java/com/github/fribourgsdp/radio/Database.kt

```diff
@@ -51,7 +51,9 @@ interface Database {
      * @return a task loading a unique ID for the user.
      */
     fun generateUserId() : Task<Long>
+    fun generatePlaylistId() : Task<Long>
```

Comment:
A short doc comment would be appreciated ;)

Reply:
Of course